types/node/scripts/generate-docs/package.json

```
@@ -0,0 +1,18 @@
+{
+  "private": true,
+  "description": "Tool to update jsdoc based on NodeJS official documentation",
+  "main": "index.js",
+  "scripts": {
+    "run": "ts-node docs.ts"
```

```suggestion
    "generate-docs": "ts-node docs.ts"
```
this will support easy to guess: `npm run generate-docs`, instead of `npm run run`